TempMail/utils/Email.swift

```diff
@@ -11,46 +11,73 @@ import SwiftUI
 
 class Email : ObservableObject{
     @Published var email_addr: String
+    @Published var inbox: [InboxModel]
     
     //  init method
     init() {
         self.email_addr = ""
+        self.inbox = [InboxModel]()
         setNewEmailAddr()
     }
     //  generates an email address
     func setNewEmailAddr() -> Void {
         //list of acceptable chars
         let letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
-        //      returns a random permutation of letters; string of up to length 9       for email addr
-        self.email_addr = String((0..<9).map{ _ in letters.randomElement()!})
+        //      returns a random permutation of letters; string of up to length 8 for email addr
+        self.email_addr = String((0..<8).map{ _ in letters.randomElement()!})
     }
     
+    // loads the Inbox that corresponds to the email address from the server, refreshing every second
+    func loadInbox() {
```

Comment:
I know it says every second, but it is actually every 10 seconds when I tested it. I was under the impression that we were just going to let it refresh in the background. If we want to go with a pull to refresh then, I'll remove the timer. Is this what we want?

Reply:
It needs to be able to do both, think of the default mail app as an example of the desired functionality

Reply:
Ok, I will work on the pull to refresh after I finish the other http get requests.

Reply:
I am going to leave it at 1 sec for now so that I can test faster, but when it gets closer to production I'll set it to 30 sec - 1 min, and maybe have it loop in a different function.